.github/workflows/authors.yml

```diff
@@ -0,0 +1,21 @@
+name: "authors update"
+on:
+  workflow_dispatch:
+
+jobs:
+  authors_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0' # this is required to actually get all the authors
+      - run: "tools/update-authors.js" # run the AUTHORS tool
+      - uses: gr2m/create-or-update-pull-request-action@v1 # create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          title: "doc: update AUTHORS"
+          body: "If this PR exists, there's presumably new additions to the AUTHORS file. This is an automatically generated PR by the `authors.yml` GitHub Action, which runs `tools/update-authors.js` and submits a new PR or updates an existing PR.\n\nPlease note that there might be duplicate entries. If there are, please remove them and add the duplicate emails to .mailmap directly to this PR."
+          branch: "actions/authors-update" # custom branch *just* for this Action.
+          commit-message: "doc: update AUTHORS"
```

Comment:
`doc` is fine but I'd prefer `meta`. The AUTHORS list really isn't documentation, at least not in the sense that a user would care about. The subsystem, as far as I know, mostly just determines where the commit ends up in the list of commits in the release notes. And I'd prefer this to be lumped in with the other meta stuff and not the doc stuff. If you disagree at all, feel free to leave it as `doc`. I'm fine with that. But if it makes no difference to you either way, I'd go with `meta` instead.

Reply:
done :)

Reply:
@bnb Looks like you changed it in the title (line 17) but not the commit-message (line 20). Can you change it in the commit message, squash it all down to one commit, and then force push? Once that's done, I'll add the commit-queue label and we can hopefully get this finally landed.

Reply:
No big deal for this, but in the future, if you could use [`--fixup` commits](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---fixupamendrewordltcommitgt) or else squash everything, it would mean we could land this by simply applying a label. If we apply the label now, the three commits will all land, and that's not really what we want.